metadata-db-app/int-test/cmr/metadata_db/int_test/concepts/cleanup_test.clj

```diff
@@ -281,40 +281,29 @@
     (is (all-revisions-exist? ta2)))
  (side/eval-form `(tk/clear-current-time!)))
 
-(deftest old-variable-revisions-are-cleaned-up
+(deftest old-variable-and-association-revisions-are-cleaned-up
   (side/eval-form `(tk/set-time-override! (tk/now)))
-  (let [variable1 (concepts/create-and-save-concept :variable "REG_PROV" 1 13)
-        variable2 (concepts/create-and-save-concept :variable "REG_PROV" 2 3)]
+  (let [coll1 (concepts/create-and-save-concept :collection "REG_PROV" 1 1)
+        coll-concept-id (:concept-id coll1)
+        variable1 (concepts/create-and-save-concept :variable "REG_PROV" 1 13 {:coll-concept-id coll-concept-id})
+        va1 {:concept-id (get-in variable1 [:variable-association :concept-id])
+             :revision-id (get-in variable1 [:variable-association :revision-id])}
+        variable2 (concepts/create-and-save-concept :variable "REG_PROV" 2 3 {:coll-concept-id coll-concept-id})
+        va2 {:concept-id (get-in variable2 [:variable-association :concept-id])
+             :revision-id (get-in variable2 [:variable-association :revision-id])}]
 
     ;; Verify prior revisions exist
-    (is (every? all-revisions-exist? [variable1 variable2]))
+    (is (every? all-revisions-exist? [variable1 variable2 va1 va2]))
```

Comment:
Separate variable associations check from the variables check and add comments to indicating the checks.